kolena/workflow/test_run.py

```diff
@@ -508,4 +508,6 @@ def test(
     :param configurations: A list of configurations to use when running the evaluator.
     :param reset: Overwrites existing inferences if set.
     """
+    if not test_suite.test_cases:
+        raise ValueError(f"test suite '{test_suite.name}' has no test cases, please add test cases to the test suite")
```

Comment:
Looks like you ended up moving away from `IncorrectUsageError` -- any particular reason?

Reply:
No particular reason, I thought these two are similar, I will use IncorrectUsageError and add a test